angularjs-portal-home/src/main/webapp/my-app/layout/services.js

```diff
@@ -96,6 +99,27 @@ define(['angular', 'jquery'], function(angular, $) {
             if (folders && 0 < folders.length) {
               result.layout = folders[0].portlets;
             }
+          } else if ($.isArray(data.layout.navigation.tabs)) { // v4-3
+            // var tabs will be tabs matching the layoutTab name
+            // expected to be an array with length 1
+            var tabs = data.layout.navigation.tabs.filter(function(el) {
+              var result = false;
+              if (el && SERVICE_LOC.layoutTab === el.name) {
+                result = true;
+              }
+              return result;
+            });
+            if (tabs && 0 < tabs.length) {
+              var columns = tabs[0].content;
+              var portlets = [];
+              function collectPortletsFromColumns(column) {
```

Comment:
just add a var declaration to this and the linter will shut up:
`var collectPortletsFromColumns = function collectPortletsFromColumns(column) {`